assets/javascripts/initializers/ai-helper.js

```diff
@@ -0,0 +1,38 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import ModalDiffModal from "../discourse/components/modal/diff-modal";
+
+function initializeProofread(api) {
+  api.addComposerToolbarPopupMenuOption({
+    action: (toolbarEvent) => {
+      const modal = api.container.lookup("service:modal");
+
+      modal.show(ModalDiffModal, {
+        model: {
+          selected: toolbarEvent.selected,
+          toolbarEvent,
```

Comment:
Why split selected out here when you are passing in the whole `toolbarEvent` object?

Reply:
yeah this modal is used in 2 spots, I can clean this up :+1: